fstat/templates/index.html

```diff
@@ -37,7 +39,7 @@
       {% for f, count in failures.most_common(total) %}
       <tr>
         <td><a href="http://git.gluster.org/cgit/glusterfs.git/tree/{{ f.signature }}" target="_blank"><code>{{ f.signature }}</code></a></td>
-        <td><a href="{{ url_for('weekly_instance_summary', num=num, fid=f.id) }}">{{ count }}</a></td>
+        <td><a href="{{ url_for('instance_summary', fid=f.id) }}">{{ count }}</a></td>
```

Comment:
I recommend passing on the date parameters for the index page as well. Otherwise, it's going to be confusing to click through. The results will change.